src/sources/soundsourcecoreaudio.cpp

```diff
@@ -4,6 +4,20 @@
 
 namespace Mixxx {
 
+namespace {
+
+// NOTE(rryan): For every seek we jump back kStabilizationFrames frames from the
+// seek position and read forward to allow the decoder to stabilize. The
+// cover-test.mp3 file needs this otherwise SoundSourceProxyTest.seekForward
+// fails. I can't find any good documentation on this on the web. Oddly, the
+// "prime" information -- which AIUI is supposed to tell us this information --
+// is zero for this file.
+static const int kStabilizationFrames = 2500;
+static CSAMPLE kStabilizationScratchBuffer[kStabilizationFrames *
```

Comment:
So wee need to fin the worst of them. An the other are suffering an unneeded performance penalty. 
Or can can we use SoundsourceMp3 here? Equal weapons for all targets, less special cased code to maintain.